asmcli/asmcli.sh

```diff
@@ -72,66 +72,63 @@ KPT_BRANCH=""
 NAMESPACE_EXISTS=0
 KUBECONFIG_SUPPLIED=0
 
-### Option variables ###
-PROJECT_ID="${PROJECT_ID:=}"
-CLUSTER_NAME="${CLUSTER_NAME:=}"
-CLUSTER_LOCATION="${CLUSTER_LOCATION:=}"
-MODE="${MODE:=}"
-CA="${CA:=}"
-KUBECONFIG="${KUBECONFIG_FILE:=}"
-CONTEXT="${CONTEXT:=}"
-
-CUSTOM_OVERLAY=""
-OPTIONAL_OVERLAY=""
-
-ENABLE_ALL="${ENABLE_ALL:=0}"
-ENABLE_CLUSTER_ROLES="${ENABLE_CLUSTER_ROLES:=0}"
-ENABLE_CLUSTER_LABELS="${ENABLE_CLUSTER_LABELS:=0}"
-ENABLE_GCP_APIS="${ENABLE_GCP_APIS:=0}"
-ENABLE_GCP_IAM_ROLES="${ENABLE_GCP_IAM_ROLES:=0}"
-ENABLE_GCP_COMPONENTS="${ENABLE_GCP_COMPONENTS:=0}"
-ENABLE_REGISTRATION="${ENABLE_REGISTRATION:=0}"
-ENABLE_NAMESPACE_CREATION="${ENABLE_NAMESPACE_CREATION:=0}"
-
-DISABLE_CANONICAL_SERVICE="${DISABLE_CANONICAL_SERVICE:=0}"
-PRINT_CONFIG="${PRINT_CONFIG:=0}"
-SERVICE_ACCOUNT="${SERVICE_ACCOUNT:=}"
-KEY_FILE="${KEY_FILE:=}"
-OUTPUT_DIR="${OUTPUT_DIR:=}"
-
-CA_CERT="${CA_CERT:=}"
-CA_KEY="${CA_KEY:=}"
-CA_ROOT="${CA_ROOT:=}"
-CA_CHAIN="${CA_CHAIN:=}"
-CA_NAME="${CA_NAME:=}"
-
-DRY_RUN="${DRY_RUN:=0}"
-ONLY_VALIDATE="${ONLY_VALIDATE:=0}"
-ONLY_ENABLE="${ONLY_ENABLE:=0}"
-VERBOSE="${VERBOSE:=0}"
-MANAGED="${MANAGED:=0}"
-MANAGED_SERVICE_ACCOUNT=""
-
-PRINT_HELP=0
-PRINT_VERSION=0
-CUSTOM_CA=0
-USE_HUB_WIP=0
-USE_VM=0
-ENVIRON_PROJECT_ID=""
-HUB_MEMBERSHIP_ID=""
-CUSTOM_REVISION=0
-WI_ENABLED=0
+### Context init JSON ###
+CONTEXT_JSON=$(cat <<EOF
```

Comment:
WDYT about putting this initial skeleton inside of context.sh instead of here? That feels like a natural place, and it should also mean we don't need to copy/paste this into the unit test file later in this PR.

Reply:
Done